internal/util/projutil/project_util.go

```diff
@@ -323,3 +324,34 @@ func PrintDeprecationWarning(msg string) {
 	fmt.Printf(noticeColor, "[Deprecation Notice] "+msg+". Refer to the version upgrade guide "+
 		"for more information: https://sdk.operatorframework.io/docs/migration/version-upgrade-guide/\n\n")
 }
+
+// RewriteFileContents adds the provided content before the last occurrence of the word label
+// and rewrites the file with the new content.
+func RewriteFileContents(filename, instruction, content string) error {
```

Comment:
Can you write a test case for this function since string manipulation is often error prone? Preferably in ginkgo/gomega.